### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/plugin/SpringBootPluginTests.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
  * @author Andy Wilkinson
  */
 @ClassPathExclusions("kotlin-daemon-client-*.jar")
-public class SpringBootPluginTests {
```
Comment (akashchandwani) [inline spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/plugin/SpringBootPluginTests.java:38]: I think we'd need a public specifier here.

Comment (izeye) [reply spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/plugin/SpringBootPluginTests.java:38]: @akashchandwani It's JUnit Jupiter-based, so I thought package-private is sufficient. Am I missing something?

Comment (akashchandwani) [reply spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/plugin/SpringBootPluginTests.java:38]: Oh Sorry, my bad...
Just checked other files in this package, they don't have `public` specifier, so I guess we'd be fine :)

**Review by akashchandwani — APPROVED** [review]

**Comment by philwebb** [comment]
Thanks once again @izeye !

**Pull request closed without merging**